measures/2024/measures-data.json

```diff
@@ -3866,7 +3941,10 @@
     "isIcdImpacted": false,
     "clinicalGuidelineChanged": [],
     "metricType": "singlePerformanceRate",
-    "allowedPrograms": [],
+    "allowedPrograms": [
+      "mips",
+      "pcf"
+    ],
```

Comment:
@peg-uehling I added PCF and MIPS to these couple of measures previously added thinking they were left out by mistake. Is that correct?

Reply:
@ckawell-sb 110 is an MVP only measure for PY23 - and I believe it is going away completely for PY24.  If so - we'll remove it when the CQM/Quality measure updates come through.

Reply:
Same for 111